specification/compute/resource-manager/readme.md

```diff
@@ -209,11 +209,35 @@ directive:
       - ACS service is deprecated so a PATCH endpoint won't be implemented
 
 ```
+### Tag: package-2020-12-01-preview
+
+These settings apply only when `--tag=package-2020-12-01-preview` is specified on the command line.
+
+``` yaml $(tag) == 'package-2020-12-01-preview'
+input-file:
+- Microsoft.Compute/stable/2020-12-01/compute.json
+- Microsoft.Compute/stable/2020-12-01/runCommands.json
+- Microsoft.Compute/stable/2019-04-01/skus.json
+- Microsoft.Compute/stable/2020-09-30/disk.json
+- Microsoft.Compute/stable/2019-12-01/gallery.json
+- Microsoft.Compute/preview/2020-10-01-preview/cloudService.json
+```
+
+### Tag: package-2020-12-01-only
+
+These settings apply only when `--tag=package-2020-12-01-preview-only` is specified on the command line.
+
+``` yaml $(tag) == 'package-2020-12-01-preview-only'
+input-file:
+- Microsoft.Compute/stable/2020-12-01/compute.json
+- Microsoft.Compute/stable/2020-12-01/runCommands.json
+```
+
 ### Tag: package-2020-10-01-preview
 
 These settings apply only when `--tag=package-2020-10-01-preview` is specified on the command line.
 
-``` yaml $(tag) == 'package-2020-10-01-preview'
+``` yaml $(tag) == 'package-2020-10-01'
```

Comment:
I do not recommend we change this tag's name, because this tag contains a preview swagger, and it should be a preview tag.

Reply:
@ArcturusZhang , this change was incorrectly made, reverting it.